app/Jobs/FetchWalletNfts.php

```diff
@@ -58,13 +66,30 @@ public function handle(): void
         );
 
         if ($result->nextToken === null) {
+            Log::info('FetchWalletNfts Job: run cleanupNftsAndGalleries', [
+                'wallet' => $this->wallet->address,
+                'network' => $this->network->id,
+                'next_token' => $result->nextToken,
```

Comment:
will always be `null` as you're inside the if case

Reply:
removed